internal/modules/clusteroverview/objects.go

```diff
@@ -111,8 +140,6 @@ var (
 		"/api-server",
```

Comment:
This one can be deleted as well. Sorry, missed this last time.

Reply:
We still need this one in order to handle links to APIServices that are still exposed in the resource viewer.